src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -174,31 +178,33 @@ void SimulatedTestFixture::runTest(
                                                         CAMERA_FRAMES_PER_AI_TICK);
 
     // Tick one frame to aid with visualization
-    bool validation_functions_done =
-        tickTest(terminating_validation_functions, non_terminating_validation_functions,
-                 simulation_time_step, ai_time_step, world);
+    bool validation_functions_done = tickTest(simulation_time_step, ai_time_step, world);
```

Comment:
Why are we able to remove the validation functions from the function call?

Reply:
they are stored as class members so they don't need to be passed in